.env.example

```diff
@@ -105,8 +105,9 @@ GITHUB_TOKEN=
 ## Secret storage
 # SECRET_STORAGE_BACKEND= # optional, should be one of: SecretStorage::DbBackend (default) or SecretStorage::HashicorpVault
 
-## Kubernets sidecar
+## Kubernets
 # SECRET_SIDECAR_IMAGE= # optional, path to the docker image that built from the secret_puller service.  service will be disabled w/o it
+# KUBERNETES_IMAGE_PULL_SECRETS="ecr-auth-us-west-1,..." # override imagePullSecrets before deploying
```

Comment:
It's not clear what this ENV var is supposed to be. Could you add a URL to some docs, or explain it?